esp/platform/sechandler.cpp

```diff
@@ -166,7 +166,9 @@ bool SecHandler::authorizeSecFeatures(StringArray & features, IEspStringIntMap &
 bool SecHandler::validateSecFeaturesAccess(MapStringTo<SecAccessFlags> & accessmap, bool throwExcpt)
 {
     StringArray features;
-    unsigned reqarray[accessmap.ordinality()];
+    unsigned reqarray[100];
```

Comment:
This feels like a backwards step to me. I think the change was made to make it compile, but there are other ways to do that (eg allocate the array dynamically...